drivers/filesystems/ext2/src/ext3/generic.c

```diff
@@ -919,6 +924,77 @@ Ext2SaveBuffer( IN PEXT2_IRP_CONTEXT    IrpContext,
 }
 
 
+#else
```

Comment:
Since the above is within #ifdef __REACTOS__ we could strip the 2empty lines 925 and 926.

Reply:
done :)